include/net/xdp.h

```diff
@@ -63,13 +63,30 @@ struct xdp_rxq_info {
 	struct xdp_mem_info mem;
 } ____cacheline_aligned; /* perf critical, avoid false-sharing */
 
+#ifdef CONFIG_XDP_LUA
+struct xdplua_create_work {
+	char				lua_script[8192];
```

Comment:
s/8192/MAXSCRIPTLEN/

Reply:
`XDPLUA_MAX_SCRIPT_LEN` or `XDPLUA_MAXSCRIPTLEN`? any variation looks good for me.. 👍